core/src/main/java/com/sequenceiq/cloudbreak/service/rdsconfig/QueryProcessorRdsConfigProvider.java

```diff
@@ -0,0 +1,60 @@
+package com.sequenceiq.cloudbreak.service.rdsconfig;
+
+import com.sequenceiq.cloudbreak.api.endpoint.v4.database.base.DatabaseType;
+import com.sequenceiq.cloudbreak.cmtemplate.CmTemplateProcessor;
+import com.sequenceiq.cloudbreak.cmtemplate.CmTemplateProcessorFactory;
+import com.sequenceiq.cloudbreak.domain.Blueprint;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Component;
+
+import javax.inject.Inject;
+
+@Component
+public class QueryProcessorRdsConfigProvider extends AbstractRdsConfigProvider {
+
+    private static final String PILLAR_KEY = "query_processor";
+
+    @Value("${cb.query_processor.database.user:query_processor}")
+    private String dasDbUser;
+
+    @Value("${cb.query_processor.database.db:query_processor}")
+    private String dasDb;
```

Comment:
I think you forget to rename the field after the copy-paste

Reply:
Acknowledged, made the changes